scripts/zones/Mount_Zhayolm/mobs/Ignamoth.lua

```diff
@@ -0,0 +1,32 @@
+-----------------------------------
+-- Area: Mount Zhayolm
+--  MOB: Ignamoth
+-----------------------------------
+require("scripts/globals/status")
+-----------------------------------
+
+function onMobSpawn(mob)
+    mob:addMod(dsp.mod.DOUBLE_ATTACK, 50)
+    mob:addMod(dsp.mod.REGAIN, 50)
+    mob:setMobMod(dsp.mobMod.ADD_EFFECT, 1)
+end
+function onMobFight(mob,target)
+end
+
+function onAdditionalEffect(mob, target, damage)
+    local dmg = math.random(300,400)
+    
+    
+    if ((math.random(1,100) >= 75) or (target:hasStatusEffect(dsp.effect.PARALYSIS) == true)) then
```

Comment:
kill extra parenthesis and strip semicolons, can omit the `== true` also

Reply:
in general this isn't right. its not applying resistance and is relying entirely on random numbers.